images-list

```diff
@@ -91,6 +91,7 @@ istio/install-cni rancher/mirrored-istio-install-cni 1.9.2
 istio/install-cni rancher/mirrored-istio-install-cni 1.9.3
 istio/kubectl rancher/istio-kubectl 1.5.10
 istio/kubectl rancher/mirrored-istio-kubectl 1.5.9
+istio/kubectl rancher/mirrored-istio-kubectl 1.5.10
```

Comment:
Whats this for again ? That image is very old.

Reply:
It appears on the images.txt of the latest rancher v2.5.8-rc12
ttps://github.com/rancher/rancher/releases/tag/v2.5.8-rc12 

and it is used here: https://github.com/rancher/system-charts/blob/7e6a27f0e377beccea51935b5af49cfc77e243b0/charts/rancher-monitoring/v0.2.2/values.yaml#L394-L395

Reply:
Ok, we can do this for now. But @aiyengar2 I think we do need long term solution here. Can we use rancher/kubectl ?

Reply:
Yup, let's swap it. This just gets used in to run `command: ["kubectl",  "apply", "-f", "/etc/operator-init/."]`:

https://github.com/rancher/system-charts/blob/7e6a27f0e377beccea51935b5af49cfc77e243b0/charts/rancher-monitoring/v0.2.2/charts/operator-init/templates/job-install-crds.yaml#L33

Reply:
@jiaqiluo there have been changes in this repo, you'll need to rebase

Reply:
@aiyengar2 @cbron  I am going to update the monitoring chart to use `rancher/kubectl:v1.20.2`. 

Should the PR for the monitoring chart go `dev-v2.5-source` or `dev-v2.6-source`?

Reply:
I'd update both so they stay in version line. Some point soon 2.5-source and 2.6-source will both have full folder on them. 
Can you also rebase this PR